Workbooks/WorkbooksMetadata.json

```diff
@@ -956,19 +956,32 @@
   "title": "Workspace audit",
   "templateRelativePath": "WorkspaceAuditing.json",
   "subtitle": "",
-  "provider": "Microsoft"
+  "provider": "Azure Sentinel community"
   },
   {
   "workbookKey": "MITREATTACKWorkbook",
   "logoFileName": "Azure_Sentinel.svg",
   "description": "Workbook to showcase MITRE ATT&CK Coverage for Azure Sentinel",
   "dataTypesDependencies": [],
   "dataConnectorsDependencies": [],
-  "previewImagesFileNames": [ "WorkspaceAuditingWhite.png", "WorkspaceAuditingBlack.png" ],
+  "previewImagesFileNames": [ "MITREATTACKWhite1.png", "MITREATTACKWhite2.png", "MITREATTACKBlack1.png", "MITREATTACKBlack2.png" ],
   "version": "1.0",
-  "title": "Workspace audit",
+  "title": "MITRE ATT&CK Workbook",
```

Comment:
Is this a planned change or just a conflict?

Reply:
this was created by a conflict that i was trying to fix in the other 2 commits, would you prefer if i close this one and submit a new PR?